content/docs/introducing-jsx.md

```diff
@@ -6,31 +6,31 @@ prev: hello-world.html
 next: rendering-elements.html
 ---
 
-Consider this variable declaration:
+დაუკვირდით ცვლადის ამ გამოცხადებას:
```

Comment:
აქ ხო არ აჯობებს ასე იყოს წინადადება ფორმულირებული:

**დაუკვირდით ამ ცვლადის გამოცხადებას**

Reply:
კი აჯობებს

Reply:
აქ მეჩვენება, რომ აქცენტი დეკლარაციაზეა და არა - ცვლადზე. შენ რას ფიქრობ?

Reply:
ქვემოთ ასეთი მაგალითია მოცემული:
const element = <h1>Hello, world!</h1>;
element-ს არ სჭირდება აქ დაკვირვება, დასაკვირვებელი აქ თითქოს არის უჩვეულო გამოცხადება. ინგლისურისთვის ბუნებრივია this variable declaration, აქ ცხადია რომ this ეხება declaration-ს და არა - variable-ს. ქართულში რომ ვწერთ ამ ცვლადის გამოცხადებას, ამ შესაძლოა ცვლადსაც მიემართებოდეს და შესაძლოა გამოცხადებასაც, ანუ აქცენტი რომელზეა, არ ჩანს. ასეთ ფორმულირებაში 'დაუკვირდით ცვლადის ამ გამოცხადებას', მკაფიოა აქცენტი. არა?

Reply:
ჩემი აზრით ცალ-ცალკე არაა, ერთ კონცეპტი აღიწერება ორი სიტყვით და მაგაზეა აქცენტი.